docs/source/dev_guide.rst

```diff
@@ -69,6 +83,97 @@ Do the same as above using the ``Makefile``.
     $ make test-all
     $ make lint
 
+
+Running notebooks tests
+-----------------------
+
+Asume the ``raven`` conda env has already been created and is up-to-date and
+raven-wps has been installed with ``make develop``.
+
+.. code-block:: console
+
+    # start local raven-wps server to test against
+    $ make start  # remember to make stop once done
```

Comment:
First of all: I really applaud the spirit of this PR! :clap: 

Here I would try to be more clear about the fact that `make start` and the other two subsequent testing commands (test all or only one notebook) are meant to be run in parallel (ideally using two consoles).

Reply:
I just tried it and realized that `make start` now starts the server in background, so it changes what I said. However I prefer to run a dev server in the foreground while testing locally, to clearly see what is happening.

Reply:
I actually prefer them (`make start` and the `make test-notebooks`) in the same console so the server-side error is aggregated/interleaved with notebook error.  In 2 different consoles, you'll have to manually match which server error is for which notebook but then the console is cleaner (no interleaving of server and notebook error).

So I think this is down to a personal preference so might not want to include that in the procedure.